build/SkiaSharp.Linux.props

```diff
@@ -0,0 +1,5 @@
+﻿<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ItemGroup>
```

Comment:
I guess that needs to be renamed to `SkiaSharp.Desktop.props`, since we might want to add some native libraries for windows here (e. g. ANGLE build for win7 runtime)